components/function-controller/.gitignore

```diff
@@ -23,3 +23,5 @@ vendor/
 *.swp
 *.swo
 *~
+
+300*.yaml
```

Comment:
If the intention is to prevent the inclusion of symlinked CRD definitons, would it make sense to clarify it as follows?
```suggestion
# vendored CRD definitions from Knative serving 
config/crds/300*.yaml
```